apps/src/util/curriculumRecommender/curriculumRecommender.js

```diff
@@ -109,6 +107,87 @@ export const getSimilarRecommendations = (
   return sortRecommendations(curriculaScores).map(curr => curr[0]);
 };
 
+// Stretch Curriculum Recommender:
+//    1) Filters out the curriculum with the a key of mainCurriculumKey, any curricula that do not support any of mainCurriculum's grade levels,
+//        and (if applicable) any curricula the user has taught before
+//    2) Scores the remaining curricula to find which is the best stretch fit for the mainCurriculum
+export const getStretchRecommendations = (
+  curriculaData,
+  mainCurriculumKey,
```

Comment:
sorry, can you explain more what "main curriculum" is?

Reply:
It's the curriculum the other curricula are being compared against to determine which one is the best similar/stretch recommendation. Happy to update the name if it's confusing!

Reply:
I can add a comment to clarify it!